examples/hpc/dla-driver/examples/dla.rs

```diff
@@ -8,8 +8,9 @@ use panic_halt as _;
 #[entry]
 fn main() -> ! {
     sprintln!("Hello world!");
-    dla_write_str("Hello DLA");
-    dla_init();
+    let mut dla = Dla::new();
+    dla.write_str("Hello DLA");
+    dla.init();
```

Comment:
This particular instance of `dla.init()` is actually a typo, since there is no method called `dla_init()` anymore. It should read `dla.init_layer()` instead. As I understand from the DLA documentation this is a mandatory step of configuring the DLA before proceeding to writing the data for the next layer. So in that sense it is not initializing DLA, even though this a necessary step before actually doing calculations. Since the function needs to be called for each layer I thought that having to explicitly call it after acquisition made sense.

Reply:
Init not merged to `Dla::new()`

Reply:
I think the per-layer API is best as discussed over lunch. This is ok for now; we can change it later.